service.yaml

```diff
@@ -49,6 +49,7 @@ modules:
   - name: github.com/getoutreach/stencil-base
   - name: github.com/getoutreach/stencil-golang
   - name: github.com/getoutreach/stencil-opslevel
+  - name: github.com/getoutreach/stencil-outreach
```

Comment:
This module is private

Reply:
Ah, artifact of running `bootstrap`, I'll remove it it's not needed.